JavaScript_4/Voice Assistant/script.js

```diff
@@ -0,0 +1,163 @@
+let personName = "";
+const toDoList = [];
+
+function greetingsUser(command) {
+  let userName = command.split(" ").slice(-1)[0];
+
+  if (!personName) {
+    personName = userName;
+    return `Nice to meet you ${userName}`;
+  } else {
+    return `You already mention your name is : ${personName}`;
+  }
+}
+
+function getName() {
+  if (personName) {
+    return `Your name is ${personName}`;
+  } else {
+    return `I dont know your name.`;
+  }
+}
+
+function addFishInMyTodo() {
```

Comment:
This function should extract an action from the command and add it to the to-do list. Implementing functions for everything people want to add to their to-do list would be daunting!